--- KindletBookletResources_es.strings	2013-01-10 14:42:31.420201845 +0100
+++ KindletBookletResources_en_GB.strings	2013-01-10 14:42:31.421201833 +0100
@@ -4,7 +4,7 @@
 Code
 LineNumberTable
 LocalVariableTable
-Lcom/amazon/kindle/kindlet/internal/KindletBookletResources_es;
+Lcom/amazon/kindle/kindlet/internal/KindletBookletResources_en_GB;
 getContents
 ()[[Ljava/lang/Object;
 <clinit>
@@ -12,28 +12,29 @@
 [Ljava/lang/Object;
 java/lang/Object
 kindlet.crash.startup.message.pattern
-Se produjo un error al intentar cerrar este contenido. Si el problema persiste, ponte en contacto con Atención al cliente de Kindle: {0}.
+An error occurred while trying to open this item. If the problem persists, please contact Kindle Customer Service at {0}.
 kindlet.crash.shutdown.message.pattern
+An error occurred while trying to close this item. If the problem persists, please contact Kindle Customer Service at {0}.
 kindlet.crash.general.message.pattern
-Se produjo un error con este contenido. Si el problema persiste, ponte en contacto con Atención al cliente de Kindle: {0}.
+An error occurred with this item. If the problem persists, please contact Kindle Customer Service at {0}.
 kindlet.remove.message
-El contenido que habías abierto ha dejado de estar disponible porque se ha eliminado del Kindle mientras este se encontraba en modo de unidad USB. Para acceder a ese contenido, vuelve a descargarlo desde Contenido archivado.
+The item that was open is now unavailable because it was removed from your Kindle while in USB drive mode. To access this item, download it again from Archived Items.
 kindlet.restart.framework.message
-El contenido no se cerró correctamente; debes reiniciar el Kindle. Si no cierras la ventana de este mensaje, el dispositivo se reiniciará en {0,number,integer} segundos.
+The item failed to close properly. Your Kindle needs to restart. Unless you close this dialogue box your device will restart in {0,number,integer} seconds.
 kindlet.restart.framework.button
-reiniciar
+restart
 kindlet.restartnow.framework.message
-Se ha producido un error con el contenido. El Kindle se reiniciará en {0,number,integer} segundos.
+An error occurred with the item. Your Kindle will restart in {0,number,integer} seconds.
 kindlet.upgrade.required.pattern
-El contenido que has intentado abrir requiere una versión más reciente del software de Kindle. Actualiza el Kindle desde Configuración o desde la página web de asistencia para dispositivos Kindle. Si necesitas más ayuda, ponte en contacto con Atención al cliente de Kindle: {0}.
+The item you attempted to open requires a newer version of Kindle software. Please update your Kindle from Settings or the Kindle Support web page. For additional help, contact Kindle Customer Service at {0}.
 kindlet.crash.lowmemory.message
-Se produjo un error con el contenido. Se cerrará el contenido y puede que se reinicie el Kindle.
+An error occurred with the item. It will close and your Kindle might restart.
 kindlet.crash.unknown
-Se produjo un error; el contenido va a cerrarse.
+An error occurred and the item will close.
 kindlet.capability.fail.pattern
-El Kindle no cumple los requisitos mínimos para procesar este contenido. Ponte en contacto con el servicio de atención al cliente de Kindle: {0}.
-com/amazon/kindle/kindlet/internal/KindletBookletResources_es
+Your Kindle does not meet the minimum requirements to run this item. Please contact Kindle Customer Service at {0}.
+com/amazon/kindle/kindlet/internal/KindletBookletResources_en_GB
 java/util/ListResourceBundle
 f
 a
-gxb
+oob
